lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,36 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    class ListCommand < HammerCLIForeman::ListCommand
+
+      output do
+        field :id, _("Id")
+        field :created_at, _("At")
+        field :remote_address, _("IP")
+        field :user_name, _("User name")
+        field :action, _("Action")
+        field :auditable_type, _("Audit type")
+        field :auditable_name, _("Audit name")
+      end
+
+      build_options
+    end
+
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
```

Comment:
My initial idea behind `audit info` was to help with audit record entries that may be very large and may not fit properly in the `audit list` output.

But your suggestion makes sense. It will be useful to indicate what changed in this audit record. Can you point me to a snippet I can use as reference?